docs/src/model_setup/output_writers.md

```diff
@@ -158,6 +158,9 @@ NetCDFOutputWriter scheduled on IterationInterval(1):
 └── file size: 17.8 KiB
 ```
 
+`NetCDFOutputWriter` also accepts an arbitrary `grid` for input that can be used to interpolate or subset the output fields. The output `grid` must be in the same architecture as the original grid.
```

Comment:
This isn't quite right.

The `grid` _must_ be consistent with output. So it's misleading to say that the output writer "accepts" an arbitrary grid. The grid is not arbitrary at all, its the opposite: the `grid` has to be consistent with `outputs`.

The right statement is something like "NetCDFOutputWriter" can also be configured for `outputs` that are interpolated or regridded to a different grid than `model.grid`. To use this functionalitt, the `output_grid` must be passed explicitly when constructing `NetCDFOutputWriter` along with the regridded / interpolated `outputs`.

Reply:
Someday this statement could be true if the `grid` is actually used to generate automatically interpolated outputs 😄 by the way (but this PR doesn't go so far)